elixir/apps/web/lib/web/live/sign_in/success.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Web.SignIn.Success do
+  use Web, {:live_view, layout: {Web.Layouts, :public}}
+
+  def mount(params, _session, socket) do
+    if connected?(socket) do
+      Process.send_after(self(), :redirect_client, 100)
+    end
+
+    query_params =
+      params
+      |> Map.take(~w[fragment state actor_name identity_provider_identifier])
+      |> Map.put(:account_slug, socket.assigns.account.slug)
+      |> Map.put(:account_name, socket.assigns.account.name)
```

Comment:
Would not break anything but it's better not to have atom and binaries as keys in the same map, will be confusing if somebody tries to debug that later :)

```suggestion
      |> Map.put("account_slug", socket.assigns.account.slug)
      |> Map.put("account_name", socket.assigns.account.name)
```

Reply:
Ah yeah, good call, will update.